tsl/test/expected/continuous_aggs_bgw.out

```diff
@@ -139,16 +139,17 @@ SELECT ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish(25);
 (1 row)
 
 SELECT * FROM sorted_bgw_log;
- msg_no | mock_time | application_name |                    msg                     
---------+-----------+------------------+--------------------------------------------
-      0 |         0 | DB Scheduler     | [TESTING] Registered new background worker
-      1 |         0 | DB Scheduler     | [TESTING] Wait until 25000, started at 0
-(2 rows)
+ msg_no | mock_time |          application_name          |                                                     msg                                                      
+--------+-----------+------------------------------------+--------------------------------------------------------------------------------------------------------------
+      0 |         0 | Continuous Aggregate Policy [1000] | materializing continuous aggregate public.test_continuous_agg_view: nothing to invalidate, new range up to 6
```

Comment:
Why do we see this extra job here now? I guess I don't expect the behavior to change when job_type was removed.

Reply:
this is because policy jobs wouldnt get executed in the scheduler test environment so their output was insert into bgw_log

Reply:
are you going to filter these again, like discussed?

Reply:
i added filter for those queries that selected from bgw_job with a job_type filter, but for bgw_log i think it's correct to include log output from the job run